build-logic/module-identity/src/main/kotlin/gradlebuild/identity/tasks/BuildReceipt.kt

```diff
@@ -93,7 +90,7 @@ abstract class BuildReceipt : DefaultTask() {
         }
         ReproduciblePropertiesWriter.store(
             Properties().apply {
-                put("commitId", commitId.getOrElse("HEAD"))
+                put("commitId", project.buildCommitId.getOrElse("HEAD"))
```

Comment:
This breaks configuration caching. Why don't you wire up the commit id as before?